src/options/index.ts

```diff
@@ -8,7 +8,7 @@ export interface Options {
   types: TypeOptions | false
   substitutionPattern: string
   configPath: string
-  schemaPath: string
+  schemaPath: string | null
```

Comment:
I've wondered about this actually. What's the best practice in TypeScript here to deal with `null` AND `undefined` at the same time?

I'm walking about the equivalent of a "Maybe" type in flow: `schemaPath: string?`

Reply:
typescript is a bit more explicit here, it's `string | null | undefined`. IIRC in function params, `?` notation can be used to mark params as optional though.